EssentialFeed/EssentialFeediOS/Image Comments UI/Controllers/ImageCommentsViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ImageCommentsViewController.swift
+//  EssentialFeediOS
+//
+//  Created by Lukas Bahrle Santana on 02/02/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public protocol ImageCommentsControllerDelegate {
+	func didRequestImageCommentsRefresh()
+}
+
+public final class ImageCommentsViewController: UITableViewController, ImageCommentsView, ImageCommentsLoadingView, ImageCommentsErrorView{
+	
+	@IBOutlet private(set) public var errorView: ErrorView!
+	
+	public var delegate: ImageCommentsControllerDelegate?
+	
+	var loaderTask:ImageCommentsLoaderTask?
```

Comment:
Missing space

```suggestion
	var loaderTask: ImageCommentsLoaderTask?
```

There are many cases like this - please, make sure the code is consistent everywhere.